app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h2>BEHOLD MY USERNESS OF ID <%=@id%></h2>
\ No newline at end of file
+<%= render @user %>
+
+<div class="bio">
+    <h2>Bio</h2>
+    <p><%=@user.bio%></p>
+</div>
+
+<%@post.each do |post|%>
+    <%= render post %>
+<%end%>
+
+<button><%= link_to ("See all posts"),user_posts_path(@user.id) %></button> 
```

Comment:
The user show page is not shown and causes an error so I am unable to review it. Kindly resolve the issue.
![user_show](https://user-images.githubusercontent.com/35504149/172709144-e087ecea-4004-40d8-a12f-d8071c821089.png)